README.md

```
@@ -195,6 +196,45 @@ LARK_E2E=1 LARK_E2E_APP_ID=… LARK_E2E_APP_SECRET=… LARK_E2E_TENANT=feishu LA
+transport。首次连接成功后的崩溃不会在运行中切换来源，而是按有界退避重启；连续健康连接
+30 秒后才重置退避。Rust 在 POSIX 上拥有整个 sidecar 进程组、在 Windows 上拥有 Job
+object，协议错误、stdout EOF、超时、关闭和 handle drop 都会终止其全部后代进程。
+
+真实 sidecar smoke 还要求操作者在连接后发送一条新私聊；未运行或 skip 不算证据：
+
+```bash
+LARK_SIDECAR_E2E=1 LARK_SIDECAR_E2E_APP_ID=… LARK_SIDECAR_E2E_APP_SECRET=… \
+  LARK_SIDECAR_E2E_TENANT=feishu \
+  cargo test --test lark_sidecar_smoke --locked -- --ignored --nocapture
+```
 ## 回复显示与 Markdown 验收
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)